prow/plugins/milestonestatus/milestonestatus.go

```diff
@@ -34,7 +34,7 @@ const pluginName = "milestonestatus"
 
 var (
 	statusRegex   = regexp.MustCompile(`(?m)^/status\s+(.+)$`)
-	mustBeSigLead = "You must be a member of the [kubernetes/kubernetes-milestone-maintainers](https://github.com/orgs/kubernetes/teams/kubernetes-milestone-maintainers/members) github team to add status labels."
+	mustBeSigLead = "You must be a member of the [%s](https://github.com/orgs/%s/teams/%s/members) github team to add status labels."
```

Comment:
I think the link needs to be `[%s/%s]` so you can pass in the team's org as another param